OrbitGl/DataView.h

```diff
@@ -94,7 +103,9 @@ class DataView {
   int sorting_column_ = 0;
   std::string filter_;
   int update_period_ms_;
-  int selected_index_;
+  std::optional<int> selected_index_;
+  absl::flat_hash_set<int> selected_indices_;
```

Comment:
I totally agree with your suggestion, i.e., keep only `selected_indices_` and use its first index for the single selection case. But this will affect a lot of places. I prefer to put it into a separated PR where we focus on unifying the single selection and multi-selection. I will create a bug in internal cleanup for this.